VMEncryption/main/check_util.py

```diff
@@ -99,6 +99,14 @@ def check_kek_url(self, test_kek_url, message):
             raise Exception('\n' + message + '\nActual: ' + test_kek_url + '\nExpected: ' + expected + "\n")
         return
 
+    def check_mhsm_url(self, test_mhsm_url, message):
+        """basic sanity check of the MHSM url"""
+        expected = "https://{managedhsm-name}.{managedhsm}.{vault-endpoint}/keys/{object-name}/{object-version}"
+        pattern = re.compile(r'^https://([a-zA-Z0-9\-]+)[\.]+(managedhsm)+([a-zA-Z0-9\-\.]+)(:443)?/keys/([a-zA-Z0-9\-]+)/([a-zA-Z0-9]+)([/]?)$', re.IGNORECASE)
```

Comment:
This regular expression matches even the successive dots. [\.]+. Is there any value in making it just [\.] ?

Reply:
Good catch, thanks. I think just [\.] is better, I will update.